src/main/utils.ts

```diff
@@ -163,7 +164,7 @@ export const loadFileFromFullPath = async (
 export const saveFileToFullPath = async (
   filePath: string,
   content: string,
-): Promise<boolean> => {
+): Promise<Result<void>> => {
   fs.writeFileSync(filePath, content);
-  return true;
+  return { ok: true, data: undefined };
```

Comment:
I would add functions (constructors) to make this type so you don't have to write this `ok: true` explicitly
e.g 
```typescript
function Ok<T>(data: T): Result<T, E> {
    return { ok: true, data }
}
function Err<E>(error: E): Result<T, E> {
    return { ok: false, error }
}
```